flixel/effects/FlxFlicker.hx

```diff
@@ -129,6 +129,16 @@ class FlxFlicker implements IFlxDestroyable
 		progressCallback = null;
 	}
 
+	public function pause():Void {
+		if (timer == null) return;
+		timer.active = false;
+	}
+
+	public function resume():Void {
+		if (timer == null) return;
+		timer.active = true;
+	}
+
```

Comment:
pretty simple change, why not, same for preventing double release, below
Made an Issue: https://github.com/HaxeFlixel/flixel/issues/3082